### PR TITLE
Cookbook view

### PR DESCRIPTION
**Pull request by shakshouka**
- coded the basic html view for the view of the cookbooks and recipes controller (index & show)
- created seeds to test my view and show functions
- added a background image for the display of the index  of the cookbooks controller
- created a partial scss file for the cookbook cards

### CODE REVIEW

**Comment by shakshouka** [comment]
handled the cloudinary storage

File: app/assets/stylesheets/components/_card_for_cookbook.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
```
Comment (OlivierCalmels) [inline app/assets/stylesheets/components/_card_for_cookbook.scss]: Fonts saved in variables in assets-stylesheets-config-font?

**Comment by shakshouka** [comment]
added in the main scss a transparency class for two colours

**Review by OlivierCalmels — APPROVED** [review]
Very good my friend!

**Pull request merged**